contributing/communications.md

```diff
@@ -17,27 +17,38 @@ Hello, so you're interested in helping out with our communications? We're pretty
 
 ## Writing Style
 
-We don't have _strict_ guidelines on how you should write. We're not going to attempt to remove your personality from your writing. This brings us onto a first key point:
+We don't have _strict_ guidelines on how you should write. We're not going to attempt to remove your personality from your writing. We have two sets of guidlines - Tone of Voice and Plain English. 
 
-### Write Like a Human...
 
-Be professional, yes, but also conversational. We don't want to sound like robots, and should certainly never be communicating on behalf of the Pa11y project in unfeeling business-speak. At all times our users should feel like they're talking to a regular human.
-
-### ...But a Nice Human
+### Tone of Voice Guidelines
 
+- Write like a human  
+Be professional, yes, but also conversational. We don't want to sound like robots, and should certainly never be communicating on behalf of the Pa11y project in unfeeling business-speak. At all times our users should feel like they're talking to a regular human.
```

Comment:
I notice you're using single line breaks here. I'm afraid they don't work :( oddly they render locally but when you push to GitHub pages they don't.

I've been using headings in lists for this, not sure whether it's correct or not but it works. [E.g. here](https://github.com/pa11y/pa11y.github.io/commit/71bad37160496cfb0965db7cc033d58c0f9a6797#diff-414a1a0a980a16f36003309fa914c17dR44)